src/control/server/init/setup_spdk.sh

```diff
@@ -5,11 +5,65 @@
 ## to be run by an unprivileged user).
 ## These sudo commands can be granted using visudo by a system administrator.
 
-set -e
+set +e
+
+thisscriptname=`basename $0`
+thisscriptpath=$(dirname $(readlink -f $0))
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Double quote to prevent globbing and word splitting. [SC2086]